### PR TITLE
CFD-247 : Group content forms : Automatically set group audience based on group context 

### PR DESCRIPTION
**Pull request by HelenaEksler**
#422

### CODE REVIEW

**Comment by amitaibu** [comment]
I see you are already using entityrefernce prepopulate here -- great.
Worth adding a behat test.

**Comment by HelenaEksler** [comment]
Would it be enough to check discussion, document and event nodes creation in tests?

**Comment by HelenaEksler** [comment]
@amitaibu Can you review this?

**Comment by amitaibu** [comment]
@HelenaEksler needs a re-roll

File: capacity4more/drupal-org.make [hunk 1]
```diff
@@ -133,6 +133,9 @@ projects[module_filter][version] = "2.0-alpha2"
 projects[node_gallery][subdir] = "contrib"
 projects[node_gallery][version] = "1.1"
 
+projects[og][download][type] = "git"
+projects[og][download][url] = "https://github.com/HelenaEksler/og.git"
```
Comment (nader77) [inline capacity4more/drupal-org.make]: Remove this.

**Comment by nader77** [comment]
@zero2one PR is ready for code review.

**Comment by zero2one** [comment]
@nader77  : Needs a re-roll

**Comment by nader77** [comment]
@HelenaEksler ^^

**Comment by zero2one** [comment]
@HelenaEksler 

Functional test : OK.

But the tests are failing:
* [Failing Behat test](https://travis-ci.org/capacity4dev/capacity4more/jobs/48473242#L1397)
* [Test takes to much time](https://travis-ci.org/capacity4dev/capacity4more/jobs/48473242#L1519)

**Comment by HelenaEksler** [comment]
@zero2one I removed these certain  tests because we don't actually need them. And also to reduce the testing time. 
About using @api instead of @javascript - I don't think so, because I already tried.

**Pull request merged**